src/lib/utils/application-commands/ApplicationCommandRegistry.ts

```diff
@@ -32,6 +32,9 @@ export class ApplicationCommandRegistry {
 	public readonly contextMenuCommands = new Set<string>();
 	public readonly guildIdsToFetch = new Set<string>();
 
+	public globalCommandId: string | null = null;
```

Comment:
Why is `globalCommands` not also a Map?

Reply:
Because each registry can only point to one global command?